cli_client/python/timesketch_cli_client/commands/analyze.py

```diff
@@ -85,13 +85,37 @@ def run_analyzer(ctx, analyzer_name, timeline_id):
 
 
 @analysis_group.command("list")
+@click.option(
+    "--output-format",
+    "output",
+    required=False,
+    help="Set output format [json, text, tabular] (overrides global setting).",
+)
 @click.pass_context
-def list_analyzers(ctx):
+def list_analyzers(ctx, output):
     """List all available analyzers.
 
     Args:
         ctx: Click CLI context object.
+        output-format: Output format to use. Available values:
+            'json','text' or 'tabular'
     """
     sketch = ctx.obj.sketch
+    if not output:
+        output = ctx.obj.output_format
+    # Show header row if output is tabular
+    if output == "tabular":
+        click.echo("Name\tDisplay Name\tIs Multi")
+
     for analyzer in sketch.list_available_analyzers():
+        if output == "json":
+            click.echo(f"{analyzer}")
+            continue
```

Comment:
Does this return one valid json dict or does this return one json dict per analyzer? If it is the former, I wonder if it would still be machine usable? (Which is the use-case for json in the first place if I'm not mistaken).

Reply:
good point. now should be better.